lib/if.c

```diff
@@ -1421,8 +1421,8 @@ static int lib_interface_destroy(struct nb_cb_destroy_args *args)
 
 	switch (args->event) {
 	case NB_EV_VALIDATE:
-		ifp = nb_running_get_entry(args->dnode, NULL, true);
-		if (CHECK_FLAG(ifp->status, ZEBRA_INTERFACE_ACTIVE)) {
+		ifp = nb_running_get_entry(args->dnode, NULL, false);
```

Comment:
`abort_if_not_found` can be `true` for `destroy()` callbacks regardless of the transaction phase (as per the API documentation).

This can be a bit confusing but it makes sense if you think about it. If a node is being deleted, it's guaranteed that its ancestors exist in the running configuration. The same can't be said about nodes being created or modified [1]. This is why there's this exception for `destroy()` callbacks.

[1] Don't forget that the NB can call the `modify()` callback for nodes that are created depending on their properties (e.g. mandatory nodes).

Reply:
@rwestphal Yes, thanks, I got it when you explained in the issue :)
But I still decided to do that, because our final goal should be to remove dependency on operational state from the validation stage and it should be easier if it doesn't have any shared code with other stages.

Reply:
Removed, because there is no shared code between validate and apply stages here.